ext/nnstreamer/tensor_decoder/tensordec-boundingbox.cc

```diff
@@ -724,7 +902,22 @@ BoundingBox::setBoxDecodingMode (const char *param)
     return FALSE;
   }
 
-  bdata = getProperties (updateDecodingMode (param));
+  const char *mode_name = updateDecodingMode (param);
+
+  if (g_strcmp0 (mode_name, "yolov8-obb") == 0) {
+    mode = YOLOV8_ORIENTED_BOUNDING_BOX;
+    bdata = getProperties (mode_name);
+  } else if (g_strcmp0 (mode_name, "yolov8") == 0) {
+    mode = YOLOV8_BOUNDING_BOX;
+    bdata = getProperties (mode_name);
+  } else if (g_strcmp0 (mode_name, "yolov5") == 0) {
+    mode = YOLOV5_BOUNDING_BOX;
+    bdata = getProperties (mode_name);
+  } else {
+    nns_loge ("Unsupported mode: %s", param);
+    return FALSE;
+  }
+
```

Comment:
Setting `mode` is needed only for the new mode. How about change like this to other bounding box modes be working:
```suggestion
  const char *mode_name = updateDecodingMode (param);
  if (g_strcmp0 (mode_name, "yolov8-obb") == 0) {
    mode = YOLOV8_ORIENTED_BOUNDING_BOX;
  }

  bdata = getProperties (mode_name);
```

We'll may need to handle setting mode later.